lib/DDG/Spice/GithubIssues.pm

```diff
@@ -14,31 +14,28 @@ spice wrap_jsonp_callback => 0;
 spice to => 'https://api.github.com/search/issues?q=$1&callback={{callback}}';
 
 my @languages = share("languages.txt")->slurp;
+@languages = sort { length($b) <=> length($a) } @languages;
+
 chomp(@languages);
 my $langs = join("|", map(quotemeta, @languages));
 
 triggers startend => 'github issue';
 
 # Handle statement
-handle query_lc => sub {
-    s/github issue//;
-    
-    if ($_ eq "") {
-        return;
-    }
-    
+handle remainder_lc => sub {
+    return unless (($_) or $_ ne "");
```

Comment:
No need for all the parens. Also if $_ is defined, it's guaranteed to have a length, so you don't need to check that its not `""` 👍 

`return unless $_;` should be fine.